var/spack/repos/builtin/packages/comd/package.py

```diff
@@ -37,20 +36,76 @@ class Comd(MakefilePackage):
     versions of CoMD will be released to incorporate the lessons learned from
     the co-design process."""
 
-    homepage = "http://exmatex.github.io/CoMD/"
+    tags = ['proxy-app']
 
-    version('master', git='https://github.com/exmatex/CoMD.git',
-            branch='master')
+    homepage = "http://www.exmatex.org/comd.html"
+    url      = "https://github.com/exmatex/CoMD/archive/master.tar.gz"
 
-    depends_on('mpi')
+    version('master', git='https://github.com/exmatex/CoMD.git', branch='master')
 
-    build_directory = 'src-mpi'
+    variant('serial', default=False, description='Build without MPI support')
+    variant('mpi', default=True, description='Build with MPI support')
+    variant('openmp', default=False, description='Build with OpenMP support')
+    variant('precision', default=True, description='Toggle Precesion Options')
```

Comment:
It isn't clear to me what this variant does. I think it would be more clear as:
```python
variant('double', default=False, description='Build with double precision instead of single')
```
or:
```python
variant('precision', default='single', values=('single', 'double'), description='Precision to build with')
```